### ISSUE TITLE
[batchnorm] Add a eps on FrozenBatchNorm2d

### ISSUE THREAD

**Issue by frgfm** [pytorch/vision#2169]
## 🚀 Feature

Introduce a `eps` attribute similar to the one of `BatchNorm2d` to enable erasing the difference in result between the two modules.

## Motivation

First, the inference does not match exactly `BatchNorm2d`:
```
import torch
from torch.nn import BatchNorm2d
from torchvision.ops.misc import FrozenBatchNorm2d

input_t = torch.rand(1, 16, 8, 8)
bn = BatchNorm2d(16).eval()
fbn = FrozenBatchNorm2d(16)
fbn.load_state_dict(bn.state_dict(), strict=False)

with torch.no_grad():
    print(torch.equal(bn(input_t), fbn(input_t)))
```
yields `False` (average absolute difference around `2.5e-6` during my tests)

Second, in `torchvision`, the implementation of `FrozenBatchNorm2d` is used only [here](https://github.com/pytorch/vision/blob/master/torchvision/models/detection/backbone_utils.py#L44) to the best of my knowledge.

If `pretrained=True` in `resnet_fpn_backbone`:
- existing `BatchNorm2d` will be replaced by `FrozenBatchNorm2d`
- the frozen BN will received parameter values from resnets trained on ImageNet **but with actual `BatchNorm2d`**.

Meaning that there might be a slightly better starting point for resnet with FPN added on top when using the pretrained version.

## Pitch

Add a `eps` attribute to `FrozenBatchNorm2d` in the constructor and in the forward method to match the implementation of `BatchNorm2d` without the batch statistics. To avoid any backward compatibility issue or discrepancies, it might have to be set to 0 by default.

Happy to come up with a PR if you think it might be worth it!

**Comment by fmassa** [comment]
Hi,

Thanks for opening the issue, there is already a PR for it in https://github.com/pytorch/vision/pull/1947 , but tests fail for some models, so this should be done with care (i.e., the default value might need to be set to zero for it to keep backwards-compatibility)

**Comment by frgfm** [comment]
Fixed by #2190

**Comment by fmassa** [comment]
Thanks @frgfm !
